### ISSUE TITLE
Pre package vulnerability DB in container images

### ISSUE THREAD

**Issue by sandman137** [deepfence/ThreatMapper#38]
Problem:

Currently, the initial vulnerability db update takes anywhere between 15-60 minutes and only after this first full update threatmapper is usable.

- Every 24 hours we can push pre-populated vulnerability DB images to threatmapper registry so when a user uses a fresh installation of threatmapper, at the very worst a 24 hr old database is being referred for scanning. This db anyway will get updated over next few hours but reduced the initial wait required.

**Comment by ramanan-ravi** [comment]
Related issue https://github.com/deepfence/dev-issues/issues/450

**Comment by ibreakthecloud** [comment]
already done, closing it.
